pydantic_ai_slim/pydantic_ai/__init__.py

```diff
@@ -2,6 +2,7 @@
 
 from .agent import Agent
 from .exceptions import AgentRunError, ModelRetry, UnexpectedModelBehavior, UsageLimitExceeded, UserError
+from .format_examples import format_examples
```

Comment:
I don't think this is so popular that it needs to be exported from `__init__.py`.

Reply:
I have removed it